packages/side-nav/src/SideNavTitle.tsx

```diff
@@ -16,27 +16,29 @@ export const SideNavTitle = ({
 }) => {
 	const Link = useLinkComponent();
 	return (
-		<Box
-			as={Link}
-			href={href}
-			padding={1}
-			color="text"
-			fontSize="sm"
-			fontWeight="bold"
-			lineHeight="default"
-			display="block"
-			css={{
-				borderBottom: `2px solid ${boxPalette.border}`,
-				textDecoration: 'none',
+		<Box as="h2">
```

Comment:
Can this just be `<H2>` from the design system? Or even just the default `h2` html element?

Reply:
Good idea! I gave it a try, but a plain `<h2>` breaks the page due to browser default margin.